app/views/issues/map.html.slim

```diff
@@ -0,0 +1,54 @@
+link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css"
```

Comment:
actually best way is to use following approach (without specifying protocol)

script src="//www.matchingnotes.com/javascripts/leaflet-google.js"